iter/filter_test.go

```diff
@@ -85,7 +85,7 @@ func TestFilterExhausted(t *testing.T) {
 }
 
 func TestExclude(t *testing.T) {
-	evens := iter.Exclude[int](iter.Count(), filters.IsEven[int])
+	evens := iter.Count().Exclude(filters.IsEven[int])
 	assert.Equal(t, evens.Next().Unwrap(), 1)
 	assert.Equal(t, evens.Next().Unwrap(), 3)
 }
```

Comment:
Don't change any of these in this file, there are testing the iterator directly.